.github/workflows/citations.yml

```diff
@@ -12,10 +12,6 @@ on:
     paths:
       - CITATION.cff
       - .github/workflows/citations.yml
-  pull_request:
-    paths:
-      - CITATION.cff
-      - .github/workflows/citations.yml
```

Comment:
cfr. previous comment: please exclude this from regular CI unless these paths have changed